src/signals_notebook/entities/container.py

```diff
@@ -59,7 +60,7 @@ def add_child(
 
         return cast(ResponseData, result.data).body
 
-    def get_children(self) -> Generator[Entity, None, None]:
+    def get_children(self, order: str = 'layout') -> Generator[Entity, None, None]:
```

Comment:
Create an enum with possible values and use it here

Reply:
Any chance we can move it to another PR?